scripts/multiqc

```diff
@@ -459,10 +439,17 @@ plots_flat, plots_interactive, make_pdf, config_file, cl_config, verbose, quiet,
                 logger.warning("Deleting    : {}   (-f was specified)".format(os.path.relpath(config.output_fn)))
                 os.remove(config.output_fn)
             else:
-                logger.error("MultiQC Report {} already exists.".format(config.output_fn))
+                report_num = 0
```

Comment:
Nitpick: I think it'd probably be nicer to start with `multiqc_report_1.html`? instead of `multiqc_report_0.html`?